pipeline/scrape-test.py

```diff
@@ -0,0 +1,71 @@
+from os import environ
+import re
+import json
+
+from dotenv import load_dotenv
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.chrome.options import Options
```

Comment:
Good spacing. I'd say the dotenv and selenium can be put together as they are both imported